evmrpc/config.go

```diff
@@ -93,6 +99,8 @@ var DefaultConfig = Config{
 	MaxTxPoolTxs:         1000,
 	Slow:                 false,
 	DenyList:             make([]string, 0),
+	MaxLogNoBlock:        10000,
+	MaxBlocksForLog:      2000,
```

Comment:
do you think we should make this larger given we have such fast blocks?  (is it disruptive to only be able to query ~11 minutes of logs?)

Reply:
i think the limit is applied "post-filter" so in most cases it would be able to cover much longer periods? For blanket filters that don't filter anything I think the recommendation from alchemy etc. is to use the subscribe endpoint instead which doesn't have such a limit